### DISCUSSION TITLE
So what's up with new firmwares?

### DISCUSSION THREAD

**Discussion by Answer-1** [gtxaspec/wz_mini_hacks#686] [General]
According to the wiki, "latest firmware" is supported for V2, but that page hasn't been updated in a while... Looking at the compile_image.sh script, firmware is 4.9.8.1002 for V2, which dates back to March 2022.
That's also when Wyze did some "security improvements" on the firmware, so did these prevented the use of later firmwares with the hack?

Just trying to confirm if my thoughts are right. Having stability issues with wyze-bridge so I wanted to try this to see if it helps.

**Comment by bjs-pdx** [comment]
**"That's also when Wyze did some "security improvements" on the firmware, so did these prevented the use of later firmwares with the hack?"**

My understanding is that the updates blocked several parts of the hack, including RTSP and others.  Wyze also changed how the wifi user name and password are stored, more recent firmware uses an encrypted file vs. plain text.

This footnote is about the V2:

**[3] Any version beyond 4.9.8.1002/4.10.8.1002 will work without audio via rtsp streaming. Pending fixes #347**

**Comment by virmaior** [comment]
i'm still running ancient firmware (4.36.2.5) but one of the main things that happened that broke something things is that wyze changed from using linked so's to putting everything all in one place. The linked so's enabled the hack to replace functions with moderate ease and more recent versions wouldn't allow for this.

there's some other issue with more recent firmwares as well.

**Comment by rspierenburg** [comment]
So both my V3s are on 4.36.11.8391 does that mean that Wz_mini_hacks won't work?  Just wondering if I should wait for an update to Wz_mini_hacks or downgrade to an earlier firmware.

Edit: Just noticed that the Firmware Support list V3 firmware that is a full 2 major updates behind.

**Reply by vic4news** [reply]
You should downgrade to firmware v4.36.9.139. Functionality is limited on higher firmware versions.

**Reply by rlust** [reply]
How do I get and load the older firmware?

**Reply by crossan007** [reply]
It looks like there's a nice repository of firmware here: https://github.com/kohrar/Wyze-Firmwares/tree/master/v3
